scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -70,15 +70,20 @@ public static BuildTargetResult CompileCoreHost(BuildTargetContext c)
             var configuration = c.BuildContext.Get<string>("Configuration");
 
             // Run the build
+            string version = DotNetCli.Stage0.Exec("", "--version").CaptureStdOut().Execute().StdOut;
+            string rid = Array.Find<string>(version.Split(Environment.NewLine.ToCharArray()), (e) => e.Contains("Runtime Id:")).Replace("Runtime Id:", "").Trim();
```

Comment:
`PlatformServices.Default.Runtime.GetRuntimeIdentifier` should get you this without having to shell out